_data/gaming.yml

```diff
@@ -203,6 +203,13 @@ websites:
     twitter: MojangSupport
     img: mojang.png
 
+  - name: Nexon America
+    url: https://www.nexon.com/main/en
+    img: nexon.png
+    tfa:
```

Comment:
You need to put the word "Yes" since you are indicating that they do support TFA.

Reply:
Sorry, I don't quite understand. Do you mean I should put a `tfa: yes`? I don't see this on other entries in the file, and in the local preview, I see it listed with the check mark in the "software" field.

![Screenshot 2019-10-15 at 21 08 26](https://user-images.githubusercontent.com/31264514/66887532-22d19380-ef90-11e9-8828-e7c5bb4906cb.png)

Reply:
You are correct, please disregard.